src/scripts/main/sidebar/index.js

```diff
@@ -338,7 +338,7 @@ const populateSidebarTertiary = (id, leafList) => {
         const newLeafContent = document.createElement("a");
 
         newLeafContent.textContent = leaf.textContent;
-        // newLeafContent.href = `#${leaf.id}`;
+        newLeafContent.href = "javascript:;";
```

Comment:
This is a little scary and is gonna be a problem for projects with CSP etc

Reply:
Wait with this until next "patch" or somehting

Reply:
This line is now commented out